### ISSUE TITLE
No module named PySide.QtCore

### ISSUE THREAD

**Issue by Conson1** [seigisato/easy_atlas#1]
First of all thank you for the tool seems incredible.
I'm getting this error:

import easy_atlas
easy_atlas.launch ()

Error: ImportError: file C: / Users / USERNAME / Documents / maya / 2018 / scripts \ easy_atlas \ core.py line 1: No module named PySide.QtCore

By the way, I'm using Maya 2018

**Comment by seigisato** [comment]
You're right, @Conson1 ! Although I've only recently uploaded this code, this tool was made a couple of years ago and it hasn't been updated since. I'll have to make different versions of it to make it compatible with different versions of Maya. Anyway, I have just created a branch with some changes that I believe will make this script work with Maya 2018 in Windows, but I am not able to test it right now. If you wish to give it a go, you will find it at the branch feature/maya_2018.

Thanks for your contribution!

**Comment by Conson1** [comment]
It worked! Thanks a lot!

**Comment by thavar** [comment]
hi, first of all many thanks for the script. it looks really promising. if I act as in the video, he does not load texture in the textureslot. and if I click make atlas, unfortunately nothing happens in the script editor. no error message, just nothing. I dont understand.

I linked the photoshop exe in the preferences. and in the fileoutput test.tga.
I have provided the texture slot with a random texture. so that there is no error message.
actually everything as in the video, only that he does not fill the texturslot and nothing happens on make atlas.
